composer.json

```
@@ -10,49 +10,47 @@
         "doctrine/doctrine-bundle": "^2.3",
         "doctrine/orm": "^2.8",
         "phpdocumentor/reflection-docblock": "^5.2",
-        "sensio/framework-extra-bundle": "^6.1",
-        "symfony/asset": "6.4.*",
-        "symfony/console": "6.4.*",
-        "symfony/dotenv": "6.4.*",
-        "symfony/expression-language": "6.4.*",
+        "symfony/asset": "7.0.*",
+        "symfony/console": "7.0.*",
+        "symfony/dotenv": "7.0.*",
+        "symfony/expression-language": "7.0.*",
         "symfony/flex": "^1.3.1",
-        "symfony/form": "6.4.*",
-        "symfony/framework-bundle": "6.4.*",
-        "symfony/html-sanitizer": "6.4.*",
-        "symfony/http-client": "6.4.*",
-        "symfony/intl": "6.4.*",
-        "symfony/mailer": "6.4.*",
+        "symfony/form": "7.0.*",
+        "symfony/framework-bundle": "7.0.*",
+        "symfony/html-sanitizer": "7.0.*",
+        "symfony/http-client": "7.0.*",
+        "symfony/intl": "7.0.*",
+        "symfony/mailer": "7.0.*",
         "symfony/mercure-bundle": "^0.3.2",
-        "symfony/messenger": "6.4.*",
+        "symfony/messenger": "7.0.*",
         "symfony/monolog-bundle": "^3.1",
-        "symfony/notifier": "6.4.*",
-        "symfony/process": "6.4.*",
-        "symfony/property-access": "6.4.*",
-        "symfony/property-info": "6.4.*",
-        "symfony/proxy-manager-bridge": "6.4.*",
-        "symfony/rate-limiter": "6.4.*",
-        "symfony/routing": "6.4.*",
-        "symfony/security-bundle": "6.4.*",
-        "symfony/serializer": "6.4.*",
-        "symfony/translation": "6.4.*",
-        "symfony/twig-bundle": "6.4.*",
-        "symfony/uid": "6.4.*",
-        "symfony/validator": "6.4.*",
-        "symfony/web-link": "6.4.*",
-        "symfony/webpack-encore-bundle": "^1.11",
-        "symfony/yaml": "6.4.*",
+        "symfony/notifier": "7.0.*",
+        "symfony/process": "7.0.*",
+        "symfony/property-access": "7.0.*",
+        "symfony/property-info": "7.0.*",
+        "symfony/rate-limiter": "7.0.*",
+        "symfony/routing": "7.0.*",
+        "symfony/security-bundle": "7.0.*",
+        "symfony/serializer": "7.0.*",
+        "symfony/translation": "7.0.*",
+        "symfony/twig-bundle": "7.0.*",
+        "symfony/uid": "7.0.*",
+        "symfony/validator": "7.0.*",
+        "symfony/web-link": "7.0.*",
+        "symfony/webpack-encore-bundle": "^2.0@dev",
```

```suggestion
        "symfony/webpack-encore-bundle": "^2.0",
```

The `@dev` feels redundant since we've set `"minimum-stability": "dev"` anyway.

---

indeed